storage/mysql/map_storage.go

```diff
@@ -270,32 +271,31 @@ func (m *mapTreeTX) LatestSignedMapRoot(ctx context.Context) (trillian.SignedMap
 
 func (m *mapTreeTX) signedMapRoot(timestamp, mapRevision int64, rootHash, rootSignatureBytes, mapperMetaBytes []byte) (trillian.SignedMapRoot, error) {
 	var rootSignature spb.DigitallySigned
-	var mapperMeta *trillian.MapperMetadata
 
 	err := proto.Unmarshal(rootSignatureBytes, &rootSignature)
 	if err != nil {
 		glog.Warningf("Failed to unmarshal root signature: %v", err)
 		return trillian.SignedMapRoot{}, err
 	}
 
-	if mapperMetaBytes != nil {
-		mapperMeta = &trillian.MapperMetadata{}
-		if err := proto.Unmarshal(mapperMetaBytes, mapperMeta); err != nil {
-			glog.Warningf("Failed to unmarshal Metadata; %v", err)
-			return trillian.SignedMapRoot{}, err
-		}
-	}
-
-	ret := trillian.SignedMapRoot{
+	smr := trillian.SignedMapRoot{
 		RootHash:       rootHash,
 		TimestampNanos: timestamp,
 		MapRevision:    mapRevision,
 		Signature:      &rootSignature,
 		MapId:          m.treeID,
-		Metadata:       mapperMeta,
 	}
 
-	return ret, nil
+	if mapperMetaBytes != nil {
+		var mapperMeta any.Any
+		if err := proto.Unmarshal(mapperMetaBytes, &mapperMeta); err != nil {
+			glog.Warningf("Failed to unmarshal Metadata: %v", err)
```

Comment:
nit: Don't log and return the error; let the caller log, if they want to.

Reply:
Done.